zig/tests/target_platform_test.bzl

```diff
@@ -112,6 +115,12 @@ def target_platform_test_suite(name):
         partial.make(_build_lib_target_platform_x86_64_windows_test, target_under_test = "//zig/tests/simple-library:library"),
         partial.make(_lib_file_extension_x86_64_linux_test, target_under_test = "//zig/tests/simple-library:library"),
         partial.make(_lib_file_extension_x86_64_windows_test, target_under_test = "//zig/tests/simple-library:library"),
+        # Test Zig target plaform on a shared library target
+        partial.make(_build_shared_lib_target_platform_x86_64_linux_test, target_under_test = "//zig/tests/simple-shared-library:library"),
+        partial.make(_build_shared_lib_target_platform_aarch64_linux_test, target_under_test = "//zig/tests/simple-shared-library:library"),
+        partial.make(_build_shared_lib_target_platform_x86_64_windows_test, target_under_test = "//zig/tests/simple-shared-library:library"),
```

Comment:
For these to work you need to add corresponding `_build_shared_lib_target...` definitions [similar to lines 60-62](https://github.com/aherrmann/rules_zig/pull/103/files#diff-a36ccfd0adf6052d9a186d49b17c840e3e6fff7fafec85bcf07f21a9e06fac13R60-R62).